language/Portuguese/strings.po

```diff
@@ -1040,1156 +1083,1182 @@ msgid ""
-msgstr ""
+msgstr "Gráficos\tImperfeitos\n"
 
 #: src/frontend/mame/ui/selgame.cpp:884
 msgid "Graphics\tOK\n"
-msgstr ""
+msgstr "Gráficos\tOK\n"
 
 #: src/frontend/mame/ui/selgame.cpp:887
 msgid "Sound\tNone\n"
-msgstr ""
+msgstr "Som\tSem Som\n"
 
 #: src/frontend/mame/ui/selgame.cpp:889
 msgid "Sound\tUnimplemented\n"
-msgstr ""
+msgstr "Som\tNão Implementado\n"
 
 #: src/frontend/mame/ui/selgame.cpp:891
 msgid "Sound\tImperfect\n"
-msgstr ""
+msgstr "Som\tImperfeito\n"
 
 #: src/frontend/mame/ui/selgame.cpp:893
 msgid "Sound\tOK\n"
-msgstr ""
+msgstr "Som\tOK\n"
 
 #: src/frontend/mame/ui/selgame.cpp:896
 msgid "Controls\tUnimplemented\n"
-msgstr ""
+msgstr "Controlos\tNão Implementados\n"
 
 #: src/frontend/mame/ui/selgame.cpp:898
 msgid "Controls\tImperfect\n"
-msgstr ""
+msgstr "Controlos\tImperfeitos\n"
 
 #: src/frontend/mame/ui/selgame.cpp:901
 msgid "Keyboard\tUnimplemented\n"
-msgstr ""
+msgstr "Teclado\tNão Implementado\n"
 
 #: src/frontend/mame/ui/selgame.cpp:903
 msgid "Keyboard\tImperfect\n"
-msgstr ""
+msgstr "Teclado\tImperfeito\n"
 
 #: src/frontend/mame/ui/selgame.cpp:906
 msgid "Mouse\tUnimplemented\n"
-msgstr ""
+msgstr "Rato\tNão Implementado\n"
 
 #: src/frontend/mame/ui/selgame.cpp:908
 msgid "Mouse\tImperfect\n"
-msgstr ""
+msgstr "Rato\tImperfeito\n"
 
 #: src/frontend/mame/ui/selgame.cpp:911
 msgid "Microphone\tUnimplemented\n"
-msgstr ""
+msgstr "Microfone\tNão Implementado\n"
 
 #: src/frontend/mame/ui/selgame.cpp:913
 msgid "Microphone\tImperfect\n"
-msgstr ""
+msgstr "Microfone\tImperfeito\n"
 
 #: src/frontend/mame/ui/selgame.cpp:916
 msgid "Camera\tUnimplemented\n"
-msgstr ""
+msgstr "Câmara\tNão Implementada\n"
 
 #: src/frontend/mame/ui/selgame.cpp:918
 msgid "Camera\tImperfect\n"
-msgstr ""
+msgstr "Câmara\tImperfeita\n"
 
 #: src/frontend/mame/ui/selgame.cpp:921
 msgid "Disk\tUnimplemented\n"
-msgstr ""
+msgstr "Disco\tNão Implementado\n"
 
 #: src/frontend/mame/ui/selgame.cpp:923
 msgid "Disk\tImperfect\n"
-msgstr ""
+msgstr "Disco\tImperfeito\n"
 
 #: src/frontend/mame/ui/selgame.cpp:926
 msgid "Printer\tUnimplemented\n"
-msgstr ""
+msgstr "Impressora\tNão Implementada\n"
 
 #: src/frontend/mame/ui/selgame.cpp:928
 msgid "Printer\tImperfect\n"
-msgstr ""
+msgstr "Impressora\tImperfeita\n"
 
 #: src/frontend/mame/ui/selgame.cpp:931
 msgid "LAN\tUnimplemented\n"
-msgstr ""
+msgstr "Rede Local\tNão Implementada\n"
 
 #: src/frontend/mame/ui/selgame.cpp:933
 msgid "LAN\tImperfect\n"
-msgstr ""
+msgstr "Rede Local\tImperfeita\n"
 
 #: src/frontend/mame/ui/selgame.cpp:936
 msgid "WAN\tUnimplemented\n"
-msgstr ""
+msgstr "WAN\tNão Implementada\n"
 
 #: src/frontend/mame/ui/selgame.cpp:938
 msgid "WAN\tImperfect\n"
-msgstr ""
+msgstr "WAN\tImperfeita\n"
 
 #: src/frontend/mame/ui/selgame.cpp:941
 msgid "Timing\tUnimplemented\n"
-msgstr ""
+msgstr "Temporização\tNão Implementada\n"
 
 #: src/frontend/mame/ui/selgame.cpp:943
 msgid "Timing\tImperfect\n"
-msgstr ""
+msgstr "temporização\tImperfeita\n"
```

Comment:
Is there a reason for not capitalising the first character of `temporização` in this string vs the one above?

Reply:
No reason, just fat fingers.
Fixed.